src/update.rs

```diff
@@ -43,27 +44,50 @@ pub struct RevealRevocationSecretKey {
 pub struct Party0 {
     x_self: OwnershipKeyPair,
     X_other: OwnershipPublicKey,
-    TX_f: FundingTransaction,
+    TX_f_body: FundingTransaction,
```

Comment:
Why the name change? What are you trying to convey?

Reply:
That the funding transaction is not signed. Maybe it's not necessary in this protocol, but I just introduced a `signed_TX_f` and renamed `TX_f` to `TX_f_body` in `create::Party6`, so I thought I would be consistent.

Reply:
An argument for this rename would be that in the update protocol we only ever use the `FundingTransaction` as an input, which is computed from the transaction body, without considering the witness stack.